src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -90,11 +90,6 @@
 helps['network application-gateway update'] = """
     type: command
     short-summary: Update an application gateway.
-    examples:
```

Comment:
Why was this simply removed?

Reply:
`--servers` is in the create command and not the update command and I did not want to create an app-gateway to confirm what a update --set command looks like. Maybe this could be a network-cli followup issue?